xml/System.Buffers.Text/Utf8Formatter.xml

```diff
@@ -20,7 +20,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides static mthods to format common data types as Utf8 strings.</summary>
```

Comment:
We should try to avoid using "utf8 string" since people might confuse it with the utf8string type we might add in the future.

Maybe just UTF-8 instead (or UTF-8 bytes/text).

```text
Provides static methods to format common data types as UTF-8.
```

**Also, note the typo:**
mthods -> methods